spec/features/users/sign_in_spec.rb

```diff
@@ -145,24 +145,24 @@ def warning_content
     end
   end
 
-  context 'signed in, session times out, sign back in', js: true do
-    it 'prompts to enter OTP' do
-      allow(Rails.application.config).to receive(:session_check_frequency).and_return(0.01)
-      allow(Rails.application.config).to receive(:session_check_delay).and_return(0.01)
-      allow(Devise).to receive(:timeout_in).and_return(1.second)
-
-      user = sign_in_and_2fa_user
-      Timecop.travel(1.minute)
-      visit '/'
-
-      fill_in 'Email', with: user.email
-      fill_in 'Password', with: user.password
-      click_button 'Log in'
-
-      expect(current_path).to eq user_two_factor_authentication_path
-      Timecop.return
-    end
-  end
+  # context 'signed in, session times out, sign back in', js: true do
+  #   it 'prompts to enter OTP' do
```

Comment:
To disable a test, just change `it` to `xit`.

Also, please include a comment with a reference to the bug.  On chromium we would do something like "TODO(sbc): Re-enable by fixing http://bug..url/"